docs/src/main/sphinx/connector/hive.rst

```diff
@@ -670,6 +670,24 @@ features:
 
 .. _hive-alter-table-execute:
 
+Overwriting data on insert
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+By default, ``INSERT`` queries are not allowed to overwrite existing data. You
+can use the catalog session property ``insert_existing_partitions_behavior`` to
+allow overwrites. Prepend the name of the catalog using the Hive connector, for
+example ``hdfs``, and set the property in the session before you run the insert
+query::
+
+    SET SESSION hdfs.insert_existing_partitions_behavior = 'OVERWRITE';
+    INSERT INTO hdfs.schema.table ...
+
+The resulting behavior is equivalent to using `INSERT OVERWRITE
+<https://cwiki.apache.org/confluence/display/Hive/LanguageManual+DML>`_ in Hive.
+
+Insert overwrite operation is not supported by Trino when the table is stored on
+encrypted HDFS or table is transactional.
```

Comment:
That's a nice update. It's bad I've missed that. 
Can you also mention that unparitioned tables are not supported as well.

Reply:
```suggestion
encrypted HDFS, when the table is transactional or when the table is unpartitioned.
```
@aczajkowski How about this?